src/VisualStudio/CSharp/Impl/LanguageService/CSharpHelpContextService.cs

```diff
@@ -153,6 +153,13 @@ private bool TryGetTextForSymbol(SyntaxToken token, SemanticModel semanticModel,
                 symbol = ((ILocalSymbol)symbol).Type;
             }
 
+            // Range variable: use the type
+            if (symbol is IRangeVariableSymbol)
+            {
+                var info = semanticModel.GetSpeculativeTypeInfo(token.SpanStart, SyntaxFactory.ParseName(symbol.Name), SpeculativeBindingOption.BindAsExpression);
+                symbol = info.Type;
```

Comment:
Wait... this is super weird.  You're taking the name, as is, and trying to parse it and then bind that?  I'm a little lost why you'd need to do this.  Won't this break badly if you don't have the right usings and whatnot?